client/packages/system/src/Clinician/ClinicianSearchInput.tsx

```diff
@@ -0,0 +1,50 @@
+import { Autocomplete } from '@openmsupply-client/common';
+import { ClinicianFragment, useClinicians } from 'packages/programs/src';
+import React from 'react';
+import { FC } from 'react';
+import { ClinicianAutocompleteOption, Clinician } from './utils';
+
+interface ClinicianSearchInputProps {
+  onChange: (clinician: ClinicianAutocompleteOption | null) => void;
+  clinicianLabel: string;
+  clinicianValue?: Clinician;
+}
+
+export const getClinicianName = (
```

Comment:
This would be good as a utility / helper function. Likely to be useful for more than just clinicians, as we have the same pattern for names generally. For now, ok to have as clinician specific tho